package.json

```diff
@@ -136,7 +136,6 @@
   "dependencies": {
     "@cardano-foundation/ledgerjs-hw-app-cardano": "1.0.7",
     "@download/blockies": "1.0.3",
-    "aes-js": "3.1.2",
```

Comment:
I think we need to delete:

`pdfjs-dist`
`blakejs`
`react-dropzone`
as well

Reply:
we still use `blakejs` for the account plates so we need it. Do we want to remove `react-dropzone` considering we've talked about adding a key import feature in the future that will require re-adding all the dropzone related code?

Reply:
gave it some thought and decided to remove dropzone. If we decide to add it again later we can look at this PR as a reference.